src/payload.config.ts

```diff
@@ -119,4 +119,9 @@ export default buildConfig({
     graphQL: {
         schemaOutputFile: path.resolve(__dirname, 'generated-schema.graphql'),
     },
+    express: {
+  		json: {
+  			limit: 1024 * 4,
```

Comment:
Increases default request payload body size. Snuck into this PR :/

Reply:
:+1: I think we will need this in any case.